### ISSUE TITLE
Unexpected line ends in v2.8.4

### ISSUE THREAD

**Issue by sebastianneubert** [BrainMaestro/composer-git-hooks#112]
I had a new behavior after updating symfony dependencies to v2.8.4.

The composer.json does not change and has the following command in there:

```
"hooks": {
            "commit-msg": [
                "# https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type",
                "rules='^(chore|docs|feat|fix|perf|refactor|style|test)(\\((\\w|[-_])+\\))?:.*'",
                "test -n \"$(grep -E $rules \"$1\")\" || {",
                "  echo >&2 \"\"",
                "  echo >&2 \"Your commit message does not follow commit message rules!\"",
                "  echo >&2 \"Please use one of these prefixes: feat fix docs style refactor perf test chore\"",
                "  echo >&2 \"\"",
```

Before the commit message was generated like this and everything works fine:
```
#!/bin/sh

# https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type
rules='^(chore|docs|feat|fix|perf|refactor|style|test)(\((\w|[-_])+\))?:.*'
test -n "$(grep -E $rules "$1")" || {
  echo >&2 ""
  echo >&2 "Your commit message does not follow commit message rules!"
  echo >&2 "Please use one of these prefixes: feat fix docs style refactor perf test chore"
  echo >&2 ""
```

After the update the script generates something like this, which results in a syntax error:
```
#!/bin/sh

# https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type && \
rules='^(chore|docs|feat|fix|perf|refactor|style|test)(\((\w|[-_])+\))?:.*'
test -n "$(grep -E $rules "$1")" || {
  echo >&2 "" && \
  echo >&2 "Your commit message does not follow commit message rules!" && \
  echo >&2 "Please use one of these prefixes: feat fix docs style refactor perf test chore" && \
  echo >&2 "" && \
```

The syntax error is caused by the new `&& \` line endings.

Edit: for better readability I shortened the complete commit message to the relevant parts.

**Comment by sebastianneubert** [comment]
I saw that the whole change from `v.2.8.3` to `v.2.8.4` is dedicated to this behavior. Unfortunately this is a breaking change in a patch version so every application with working hooks will break here.

If you assume, that every line in the hooks script is a single command, your solution is okay. But then I have to rewrite my whole script in one line which is very ugly in the composer.json.

Maybe you could provide a solution in a new major version, where the single commands aren't only strings and can also be arrays without these `&& \`

Example:
```
{
    "extra": {
        "hooks": {
            "pre-commit": [
                [
                    "echo this is a",
                    "echo multiline command"
                ]
                "php-cs-fixer fix ." // fix style
            ],
            "...": "..."
        }
    }
}
```

**Comment by BrainMaestro** [comment]
@sebastianneubert this should be fixed now in `v2.8.5`, also there will be a proper solution for concatenating commands in the next feature release. I'll close this, but let me know if you encounter any issues
